src/FormDecorator/IcingaFormDecorator.php

```diff
@@ -73,16 +75,23 @@ protected function createCheckbox(CheckboxElement $checkbox)
     protected function assembleLabel()
     {
         $label = parent::assembleLabel();
-        if (! $this->formElement instanceof FieldsetElement) {
-            if ($label !== null) {
-                $label->addWrapper(new HtmlElement('div', Attributes::create(['class' => 'control-label-group'])));
-            } else {
-                $label = new HtmlElement(
-                    'div',
-                    Attributes::create(['class' => 'control-label-group']),
-                    HtmlString::create('&nbsp')
-                );
-            }
+
+        if (
+            $this->formElement instanceof FieldsetElement ||
+            $this->formElement instanceof SubmitElement ||
+            $this->formElement instanceof SubmitButtonElement
```

Comment:
There's the `FormSubmitElement` interface. Use it instead and the `if` doesn't need to be disassembled. You're introducing way to much noise in the change history with this litte change.